include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -2715,6 +2765,22 @@ class FEFaceEvaluation : public FEEvaluationAccess<dim,
    */
   const unsigned int dofs_per_cell;
 
+  /**
+   * The number of degrees of freedom of a single component on the cell for
+   * the underlying evaluation object. Usually close to
+   * static_dofs_per_component, but the number depends on the actual element
+   * selected and is thus not static.
+   */
+  const unsigned int dofs_per_component_on_face;
+
+  /**
+   * The number of degrees of freedom on the cell accumulated over all
+   * components in the current evaluation object. Usually close to
+   * static_dofs_per_cell = static_dofs_per_component*n_components, but the
+   * number depends on the actual element selected and is thus not static.
+   */
+  const unsigned int dofs_per_face;
```

Comment:
I would not choose these names, and choose something more specific to indicate how to interpret this number and avoid confusion with `FiniteElement::dofs_per_face` which has a different meaning.